R/assocTestSingle.R

```diff
@@ -93,10 +106,25 @@ setMethod("assocTestSingle",
 setMethod("assocTestSingle",
           "GenotypeIterator",
           function(gdsobj, null.model, test=c("Score", "Score.SPA", "BinomiRare", "CMP"),
-                   recalc.pval.thresh=0.05, GxE=NULL,
+                   recalc.pval.thresh=0.05, approx.score.SE=FALSE, GxE=NULL,
```

Comment:
take out the `approx.score.SE=FALSE` argument from this method since it's not implemented. methods for different objects can have different sets of arguments.

Reply:
done